api_statistic_test.go

```
@@ -0,0 +1,68 @@
+		DataUsage:              0,
+		GrainsCount:            0,
+		GrainsCountSum:         0,
+		GrainsCountConsume:     0,
+		InteractionPeopleCount: 34,
+		ContentCount:           1806,
+		CollaborationCount:     97013,
+		WorkingHours:           38870,
+		Baike:                  1.7,
+		TableCount:             3,
+	}, resp)
+
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed

(gofumpt)